webapp/src/components/MeetingAccordion.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Link as MuiLink,
+  ListItem,
+  ListItemText,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import React from 'react';
+
+import { formatDate, formatTime } from '../helpers/dateTime';
+import { Meeting as MeetingType } from '../types/api';
+import Meeting from './Meeting';
+
+interface MeetingAccordionProps {
+  meeting: MeetingType;
+}
+
+const MeetingAccordion = ({ meeting }: MeetingAccordionProps) => {
+  const [isInDrawer, setisInDrawer] = React.useState(true);
```

Comment:
@pthan1 settisInDrawer is never used. If it's not needed maybe we should get rid of it.

Reply:
It's not needed, I'll take it out.